packages/build/src/commands/run.js

```diff
@@ -125,7 +125,14 @@ const runCommand = async function({
   timers,
   testOpts,
 }) {
-  if (!shouldRunCommand({ event, package, error, failedPlugins })) {
+  if (
+    !shouldRunCommand({
+      event,
+      package,
+      error,
+      failedPlugins,
+    })
+  ) {
```

Comment:
You might need to run `npm run format` to run Prettier on this PR (unless the former code here was not prettified).